packages/airnode-abi/package.publish.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@api3/airnode-abi",
+  "license": "MIT",
+  "version": "0.1.0",
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./dist",
+    "compile": "tsc -p tsconfig.build.json"
+  },
+  "dependencies": {
+    "lodash": "^4.17.20"
```

Comment:
Is it necessary to add lodash here? This is also available in the root package.json

Reply:
Yeah because the one in the root package.json is not included in the independently published package. That's essentially why a separate package.publish.json is needed